### ISSUE TITLE
'KeyMissing', 'Secret key are required'

### ISSUE THREAD

**Issue by just-a-normal-human** [Binance-docs/Binance_Futures_python#49]
The error ```BinanceApiException: ('KeyMissing', 'Secret key are required')``` arises even tho I've entered my keys in ```RequestClient.py``` .  I'm surely just missing something simple. How is ```__init__()```supposed to look with correctly entered keys?

Thanks for your work!

**Comment by vanshwassan** [comment]
You are not supposed to add your keys in requestclient.py or subscriptionclient.py. Just add the keys in your script like - 
`request_client = RequestClient(api_key="api_key", secret_key="secret_key")`
and remove the comment from binance_f/constant/system.py and binance_d/constant/system.py to enable fapi.binance.com instead of testnet

**Comment by just-a-normal-human** [comment]
Yeah, I noticed that and made a seperate ```privateconfig.py``` file and loaded them to ```test.py```. Which made everything work!

When i tried inserting my keys in my script i got an error that the keys are in the wrong format. I am however very new to this so i might've done something wrong but the first method worked.

Thanks for your answer!
